Formula/dgraph.rb

```diff
@@ -0,0 +1,63 @@
+class Dgraph < Formula
+  desc "Fast, Distributed Graph DB"
+  homepage "https://dgraph.io"
+  url "https://github.com/dgraph-io/dgraph/archive/v20.03.1.tar.gz"
+  sha256 "abe217853134d86b9dba2814d5ef521730cc9eb9c6e0825658cc62af2562b363"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = bin
+    ENV["GOPATH"] = buildpath/"build"
```

Comment:
Do we need these two options? I saw the upstream repo does have go modules support (which means we can remove `GOPATH` env)?

Reply:
Nice catch. `GOPATH` can be removed, but get `Error: Empty installation` without `GOBIN`.

Reply:
perfect!